src/components/src/VRProductDisplayItem/README.md

```diff
@@ -12,6 +12,12 @@ For an example of how to integrate this component into your own project checkout
 
 `productdisplayitem.main.tsx`
 
+The component gives the user the ability to rotate in the vr-scene with mobile, dedicated vr devices and on desktop.  Look at the aframe website for a more exhaustive list.  
+The component also allows the user to set a panorama image, a 3D mesh, and an html markup that populates a 3D plane.  
+
+Note that the html markup that populates the plane in the 3d scene
+is hidden outside of aframes `Scene` tag.  Populate markup inside the div with the classname `inner-html-texture` to customize desired information to be shown in the 3D scene.
```

Comment:
Unnecessary new line.

Reply:
I've reworded it.  The newline seems to make sense to me.  Let me know if it still doesn't